stylesheets/mainpagestyle.css

```diff
@@ -0,0 +1,550 @@
+.no-style {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Lato, sans-serif;
+  font-size: 16px;
+}
+
+.control-bar {
+  display: none;
+}
+
+.hamburger-wrapper {
+  position: absolute;
+  left: 5%;
+  top: 5%;
+}
+
+.navigation-bar {
+  display: none;
+}
+
+.logo-image {
+  display: none;
+}
+
+.section-intro {
+  padding: 25% 5%;
+  background: #f7f7f7 url('../bg-icons/main-page-bg.svg') 0 0 no-repeat;
+  background-size: contain;
+}
+
+.header-welcome,
+.header-title {
+  color: #ec5242;
+  background-color: #f7f7f7;
+}
```

Comment:
- [ ] You can make your header texts more appealing by giving it a background image and using ` -webkit-background-clip: text;` &`  -webkit-text-fill-color: transparent;` properties to match the original design

![image](https://user-images.githubusercontent.com/98453979/210373390-84d9208b-bcd2-4a0e-8ca4-9c629ec78255.png)

### Checkout the original design

![image](https://user-images.githubusercontent.com/98453979/210373599-5a58d684-100a-4278-ad99-4d57b2796343.png)